aio_pika/message.py

```diff
@@ -645,12 +649,22 @@ def __init__(self, message: IncomingMessage, *,
 
         if not self.ignore_processed or not self.message.processed:
             if self.reject_on_redelivered and self.message.redelivered:
-                log.info("Message %r was redelivered and will be rejected",
-                         self.message)
-
-                await self.message.reject(requeue=False)
+                if not self.message.channel.is_closed:
+                    log.info(
+                        "Message %r was redelivered and will be rejected",
+                        self.message
+                    )
+                    await self.message.reject(requeue=False)
+                else:
+                    log.info(
```

Comment:
It's should be `log.warning`, so it looks important.

Reply:
@mosquito I've applied the changes you've suggested.